# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [1]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [7]:
series1 = pd.Series(np.random.randint(1, 5, size = 100))
series2 = pd.Series(np.random.randint(1, 4, size = 100))
series3 = pd.Series(np.random.randint(10000, 30001, size = 100))

### Step 3. Let's create a DataFrame by joinning the Series by column

In [10]:
df = pd.concat([series1, series2, series3], axis = 1)

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [11]:
df.columns = ["bedrs", "bathrs", "price_sqr_meter"]

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [14]:
bigcolumn = pd.concat([series1, series2, series3], axis = 0).to_frame()
bigcolumn

0
0       1
1       2
2       3
3       3
4       3
..    ...
95  17122
96  28364
97  26028
98  24431
99  25466

[300 rows x 1 columns]

### Step 6. Oops, it seems it is going only until index 99. Is it true?

In [15]:
bigcolumn.reset_index(drop = True)

0
0        1
1        2
2        3
3        3
4        3
..     ...
295  17122
296  28364
297  26028
298  24431
299  25466

[300 rows x 1 columns]

### Step 7. Reindex the DataFrame so it goes from 0 to 299